pkg/kubelet/container/runtime.go

```diff
@@ -94,6 +94,8 @@ type Runtime interface {
 	RemoveImage(image ImageSpec) error
 	// Returns Image statistics.
 	ImageStats() (*ImageStats, error)
+	// Returns the filesystem path of the pod's network namespace
+	GetNetNS(containerID ContainerID) (string, error)
```

Comment:
I don't really see the difference right now between GetNetNS() and a NetworkPluggableRuntime.  If we add more methods that runtimes won't implement, then sure, maybe we re-evaluate.

Also note that the only reason there *is* a GetRuntime() is for GetNetNS(); CNI uses it for Status() too but I think that's bogus and it should follow the same model as kubenet does, but that's another PR.  I think we can probably get rid of GetRuntime() in favor of some other interface that the runtime actually implements.  Even better, since the runtimes themselves are calling the functions, we should just pass an interface into them (or nil) instead of having a roundable GetRuntime().

In summary, I'd like to go with GetNetNS() for now, and rework GetRuntime() in a further PR if that's OK?  With the understanding that yes, not all runtimes support returning a netns path now or in the future.